docs/source/config_file.rst

```diff
@@ -529,6 +529,12 @@ Miscellaneous strictness flags
     Allows variables to be redefined with an arbitrary type, as long as the redefinition
     is in the same block and nesting level as the original definition.
 
+.. confval:: disable_error_code
+
+    :type: comma-separated list of strings
+
+    Causes mypy to suppress errors with given error codes.
```

Comment:
One nit: I'd steal the language from the command line docs here (for consistency, and for the mention of "globally").
```suggestion
    Allows disabling one or multiple error codes globally.
```

(More generally, duplication between command line docs and config docs seems like a bit of a maintenance headache)